cmd/doracled/cmd/register_oracle.go

```diff
@@ -60,13 +67,11 @@ func registerOracleCmd() *cobra.Command {
 			// sign and broadcast to Panacea
 			msgRegisterOracle := oracletypes.NewMsgRegisterOracle(uniqueID, oracleAccount.GetAddress(), nodePubKey, nodePubKeyRemoteReport, trustedBlockInfo.TrustedBlockHeight, trustedBlockInfo.TrustedBlockHash)
 
-			cli, txBuilder, err := generateGrpcClientAndTxBuilder(conf)
+			txBuilder := panacea.NewTxBuilder(*queryClient)
+			cli, err := panacea.NewGrpcClient(conf)
 			if err != nil {
-				return fmt.Errorf("failed to generate gRPC client and/or Tx builder: %w", err)
+				return fmt.Errorf("failed to generate node key pair: %w", err)
```

Comment:
```suggestion
				return fmt.Errorf("failed to generate gRPC client: %w", err)
```

Reply:
Thanks, I fixed it in [f7b488e](https://github.com/medibloc/panacea-doracle/pull/46/commits/f7b488e1dd94f4dfc16eac4e9b5036b9ba09d41c).